pinc/theme.inc

```diff
@@ -622,38 +622,21 @@ function show_tally_specific_stats($tally_name)
         // The requestor is a logged-in user.
         // Show the user's personal statistics
 
-        //Get the personal statistics array
-        $neighbors =
-            user_get_page_tally_neighborhood(
-                $tally_name,
-                $user->username,
-                $user->u_neigh
-            );
-        $usern = $neighbors[0];
+        $current_tally = $tallyboard->get_current_tally($user->u_id);
+        $current_rank = $tallyboard->get_rank($user->u_id);
 
         //get rank
-        global $testing;
-        if ($testing) {
```

Comment:
Removing this was intentional. The source code is publicly accessible -- no one needs to inflate pagecounts on TEST to get a list of them.